app/assets/javascripts/dashboard.js

```diff
@@ -79,8 +80,11 @@ MinionPoller = {
         for (i = 0; i < minions.length; i++) {
           rendered += MinionPoller[renderMethod].call(MinionPoller, minions[i]);
 
-          if (minions[i].highstate != "applied") {
-            allApplied = false;
+          if (minions[i].highstate == "applied") {
+            allApplied = true;
```

Comment:
i just realized we don't even use allApplied anymore... i think we can just drop it